slothy/core/heuristics.py

```diff
@@ -584,50 +697,64 @@ def _split_inner(body, logger, conf, ssa=False):
                 body = Heuristics._get_ssa_form(body, logger, conf)
                 Heuristics._dump("Code in SSA form:", body, logger, err=True)
 
-            body, perm = Heuristics._naive_reordering(body, log, conf,
-                use_latency_depth=conf.split_heuristic_preprocess_naive_interleaving_by_latency)
+            body, perm = Heuristics._naive_reordering(
+                body,
+                log,
+                conf,
+                use_latency_depth=(
+                    conf.split_heuristic_preprocess_naive_interleaving_by_latency,
```

Comment:
Breaks the naive interleaving. 

```suggestion
                    conf.split_heuristic_preprocess_naive_interleaving_by_latency
```